src/components/shared/Tray/TrayComponents/SelectedService/SelectedService.js

```diff
@@ -1,35 +1,36 @@
-import React, { useContext } from 'react';
-import { FetchDisruptionsContext, AutoCompleteContext } from 'globalState';
+import React from 'react';
 // Import customHooks
-import useFilterLogic from 'customHooks/useFilterLogic';
-// Imported components
 import Message from 'components/shared/Message/Message';
+import useShowSelectedServicesInfo from './customHooks/useShowSelectedServiceInfo';
+// Imported components
 import DisruptedService from './DisruptedService/DisruptedService';
 import InfoAboutSelectedService from './InfoAboutSelectedService/InfoAboutSelectedService';
 import SaveRoutesMessage from './SaveRoutesMessage/SaveRoutesMessage';
 
 const SelectedService = () => {
-  const [fetchDisruptionsState] = useContext(FetchDisruptionsContext);
-  const [autoCompleteState] = useContext(AutoCompleteContext);
-  // The below will check all disruptions and will return any disruption where the mode is bus and the id the user clicked in the autocomplete is within the servicesAffected array
-  const selectedData = useFilterLogic();
+  const {
+    showInfoAboutSelectedService,
+    showServiceMessage,
+    showDisruptedServices,
+    showLineBreak,
+    disruptedServices,
+  } = useShowSelectedServicesInfo();
 
   return (
     <>
       {/* If ann item has been selected that is not by the map then show selected services that you can fav and also show save message box. selectedItem.severity is used as the if statement to ensure all other data has been mapped to the state, using the id would cause an error as this can be gotten from the URL */}
-      {autoCompleteState.selectedItem.severity && !autoCompleteState.selectedItem.selectedByMap && (
+      {showLineBreak && <hr className="wmnds-col-1" />}
+      {showInfoAboutSelectedService && (
         <>
-          <hr className="wmnds-col-1" />
           <SaveRoutesMessage />
           <InfoAboutSelectedService />
         </>
       )}
       {/* If no selectedData then it must be good service */}
-      {!selectedData.length && <Message />}
+      {showServiceMessage && <Message />}
       {/* If there are selectedData then there must be disruptions, loop through */}
```

Comment:
The purple "save routes" message box no longer appears when clearing localstate.

How it (should) looks on live site:
![image](https://user-images.githubusercontent.com/13015201/102202610-72c32780-3ebf-11eb-8b65-e7fcf1c65bc1.png)


How yours looks:
![image](https://user-images.githubusercontent.com/13015201/102202668-88d0e800-3ebf-11eb-8f6b-7d92a40cabec.png)

Reply:
Closing...none-issue.